2200041/10505.py

```diff
@@ -0,0 +1,17 @@
+#10505
+# T, 사람, 소득, 평균, 저소득(초기화)
+# 평균보다 작은 사람구함
+
+T = int(input())
+
+for t in range(1, T+1):
+    low_pay = 0
+    peope = int(input())
+    pay = list(map(int, input().split()))
+    avg = sum(pay) // peope
```

Comment:
```suggestion
    avg = sum(pay) / peope
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄